doc/user/content/architecture/arrangements.md

```diff
@@ -3,6 +3,10 @@ title: "Arrangements"
 description: "Understand how Materialize arrangements work."
 aliases:
   - /overview/arrangements/
+  - /get-started/arrangements/
+menu:
+  main:
+    parent: architecture
```

Comment:
Didn't read this page  ... just bubbling up in nav as a first step.  But, per comment, https://github.com/MaterializeInc/materialize/pull/31201#discussion_r1938298247, I think it's mostly accurate.